internal/version/version.go

```diff
@@ -1,9 +1,19 @@
 package version
 
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"runtime/debug"
+)
+
+const smHomepage = "https://grafana.com/docs/grafana-cloud/synthetic-monitoring/"
```

Comment:
nit: let's link to this repo because grafana.com paths can change without notice.

```suggestion
const smHomepage = "https://github.com/grafana/synthetic-monitoring-agent"
```